docs/deployment/helm.md

```diff
@@ -177,7 +177,7 @@ We have introduced the following Chart parameters.
 | `topologyUpdater.affinity`                    | dict   | {}                      | Topology updater pod [affinity](https://kubernetes.io/docs/tasks/configure-pod-container/assign-pods-nodes-using-node-affinity/)                                                                      |
 | `topologyUpdater.config`                      | dict   |                         | [configuration](../reference/topology-updater-configuration-reference)                                                                                                                                |
 | `topologyUpdater.podSetFingerprint`           | bool   | false                   | Enables compute and report of pod fingerprint in NRT objects.                                                                                                                                         |
-| `topologyUpdater.kubeletStateDir`             | string | "/host-var/lib/kubelet" | Specifies kubelet state directory path for watching state and checkpoint files. Empty value disables kubelet state tracking.                                                                          |
+| `topologyUpdater.kubeletStateDir`             | string | /host/lib/kubelet | Specifies kubelet state directory path for watching state and checkpoint files. Empty value disables kubelet state tracking.                                                                          |
```

Comment:
The default value is `/host-var/lib/kubeler` i think. Check here:
https://github.com/kubernetes-sigs/node-feature-discovery/blob/master/cmd/nfd-topology-updater/main.go#L44

Reply:
Argh, unbelievable fat fingering. It's `/var/lib/kubelet`, we just changed that in #1207 

Thanks for pointing this out @PiotrProkop. It's fixed now.